docs/Forms.md

```diff
@@ -0,0 +1,142 @@
+# Forms
+
+BTCPay Server's Forms feature enables you to request that your customer complete a form before proceeding with a payment.
+
+These forms are fully customizable to suit your requirements.
+
+Example of form definition:
+
+```json
+{
+  "fields": [
+    {
+      "name": "buyerEmail",
+      "constant": false,
+      "type": "email",
+      "value": null,
+      "required": true,
+      "label": "Enter your email",
+      "helpText": "This is help text",
+      "fields": []
+    },
+    {
+      "name": "buyerName",
+      "constant": false,
+      "type": "text",
+      "value": null,
+      "required": true,
+      "label": "Name",
+      "helpText": null,
+      "fields": []
+    },
+    {
+      "name": "buyerAddress1",
+      "constant": false,
+      "type": "text",
+      "value": null,
+      "required": true,
+      "label": "Address Line 1",
+      "helpText": null,
+      "validationErrors": [],
+      "fields": []
+    },
+    {
+      "name": "buyerAddress2",
+      "constant": false,
+      "type": "text",
+      "value": null,
+      "required": false,
+      "label": "Address Line 2",
+      "helpText": null,
+      "fields": []
+    },
+    {
+      "name": "buyerCity",
+      "constant": false,
+      "type": "text",
+      "value": null,
+      "required": true,
+      "label": "City",
+      "helpText": null,
+      "fields": []
+    },
+    {
+      "name": "buyerZip",
+      "constant": false,
+      "type": "text",
+      "value": null,
+      "required": false,
+      "label": "Postcode",
+      "helpText": null,
+      "fields": []
+    },
+    {
+      "name": "buyerState",
+      "constant": false,
+      "type": "text",
+      "value": null,
+      "required": false,
+      "label": "State",
+      "helpText": null,
+      "fields": []
+    },
+    {
+      "name": "buyerCountry",
+      "constant": false,
+      "type": "text",
+      "value": null,
+      "required": true,
+      "label": "Country",
+      "helpText": null,
+      "fields": []
+    }
+  ]
+}
+```
+
+Output:
+
+![Form](./img/Forms-1.png "Form")
+
+In a field definition, only the following fields can be set:
+
+| Field   |      Description      |
+|----------|-------------|
+`.fields.constant` | If `true`, the `.value` must be set in the form definition, and the user will not be able to change the field's value. ( example: the form definition's version) |
```

Comment:
```suggestion
|`.fields.constant` | If `true`, the `.value` must be set in the form definition, and the user will not be able to change the field's value. ( example: the form definition's version) |
```

Hopefully that fixes the problematic table formating in the [preview](https://deploy-preview-1251--btcpayserver-docs-preview.netlify.app/Forms/):
![grafik](https://user-images.githubusercontent.com/1374810/227867300-55caf741-fd4c-4c76-a8af-9d65ddafe7ba.png)

Reply:
I don't think it does, but I can check if we can improve it with some CSS.